### ISSUE TITLE
 Add new display GDEY0213B74. 250*122 4 grays

### ISSUE THREAD

**Issue by martinberlin** [martinberlin/cale-idf#68]
Product: https://www.good-display.com/product/391.html
Check datasheets. This one is not on any other know component.
WiKi: https://github.com/martinberlin/cale-idf/wiki/Good-Display-gdey0213b74

Got the right LUTs for 4 grays. 
Problem of partial update not solved (yet)

**Comment by martinberlin** [comment]
@Frreddie I managed to do already the monochrome update. 

Tried partials with same RamSettings and the way the older gdeh0213b73 class was done but it does not work the same:
![Partial update](https://user-images.githubusercontent.com/2692928/198663979-a6a96542-2967-4e84-b91a-c0e506d7d669.jpg)

It's doing the partial without flashing but is copying the contents starting from the marked line btw. GOOD DISPLAY and doing it like 50 pixels below. The circle drawn is correct but is also shifted. 
DEBUG Parameters

_SetRamArea(xS:0,xE:15,Ys:100,Y1s:0,Ye:159,Ye1:0)
_SetRamPointer(addrX:0,addrY:100,addrY1:0)
Loop from ys:100 to ye:159 -> and sends that area of the framebuffer (This seems correct)

What it's wrong IMHO is the RAM pointing. For this I will need some help. 
Then I will add the grays following my working class for 4.2 display.
